src/cares_wrap.cc

```diff
@@ -296,6 +300,93 @@ Local<Array> HostentToNames(Environment* env, struct hostent* host) {
   return scope.Escape(names);
 }
 
+void safe_free_hostent(struct hostent* host) {
+  int idx;
+
+  if (host->h_addr_list) {
```

Comment:
Just a style nit, but we do prefer explicit `!= nullptr` or `== nullptr` for these checks